.vscode/settings.json

```diff
@@ -23,5 +23,20 @@
     },
     "cSpell.words": [
         "Daffio"
+    ],
+    "editor.codeActionsOnSave": {
```

Comment:
This is invalid JSON. Should be:
```json
"editor.codeActionsOnSave": {
    "source.fixAll.eslint": true
},
```

Reply:
Yep -- I messed up a rebase here. My intent was to leave that particular setting up to devs who want it, but the other's are QoL improvements.